modules/theme/plugin.js

```diff
@@ -0,0 +1,44 @@
+import Vue from 'vue'
+import { parse, serialize } from 'cookie'
+
+const key = '<%= options.cookie.key %>'
+const cookieOptions = <%= JSON.stringify(options.cookie.options, null, 2) %>
+
+export default function (ctx, inject) {
+  const theme = Vue.observable({
+    value: (process.client ? window.__nuxt_theme : '<%= options.value %>')
```

Comment:
- Can we just make `value` observable? Assigning a function to a reactive object is little bit hacky and confusing when one tries to debug `$nuxt.$theme`
- The field name of `value` for us makes sense but probably little bit confusing for a general variable naming. What about `$theme.name` or `$theme.variant` (also same for nuxt config)

Reply:
Or `theme.mode`. I actually call this `color modes` in my apps, since `theme` for me is more like a template. But it's purely naming issue.